tests/unit/test_charm.py

```diff
@@ -571,13 +607,19 @@ def test_given_dns_config_contains_non_ip_when_install_then_status_is_blocked(
             captured.records[0].getMessage(),
         )
 
+    @patch(
+        "test_charm.MagmaAccessGatewayOperatorCharm._is_magmad_enabled", new_callable=PropertyMock
+    )
     @patch("subprocess.run")
     @patch("netifaces.interfaces")
     def test_given_valid_dhcp_config_when_install_then_status_is_active(
```

Comment:
The test name doesn't match what we validate

Reply:
Fixed